desktop/libs/indexer/src/indexer/indexers/rdbms.py

```diff
@@ -175,10 +175,24 @@ def run_sqoop(request, source, destination, start_time):
       rdbms_port = source['rdbmsPort']
       rdbms_user_name = source['rdbmsUsername']
       rdbms_password = source['rdbmsPassword']
+      rdbms_driver = source['rdbmsType']
       url = rdbms_host
-
+      if rdbms_driver != 'jdbc':
```

Comment:
Nit: Let's add a comment why we needed this?

Reply:
rdbms_driver could be 'jdbc', 'mysql', 'postgresql' or 'oracle'.
```python
rdbms_driver = source['rdbmsType']
```
* When rdbms_driver is jdbc, url have jdbc scheme because Importer UI requires user to enter url with jdbc scheme like below.
![image](https://user-images.githubusercontent.com/53476980/232223477-18cd5c81-ca53-496c-8ab4-645050068bec.png)

---
* However, rdbms_driver is mysql, 'postgresql' or 'oracle'.
Importer UI doesn't require user to do that. it requires to enter just hostname and port number.
![image](https://user-images.githubusercontent.com/53476980/232223552-8158f54b-ae0d-4c6e-b2d1-822fb7795f5d.png)

---
This is the problem. run_sqoop function make sqoop command using 'url' variable.
```python
statement = '--connect %(url)s/%(rdbmsDatabaseName)s --username %(rdbmsUserName)s --password-file %(passwordFilePath)s' % {
      'url': url,
      'rdbmsDatabaseName': rdbms_database_name,
      'rdbmsUserName': rdbms_user_name,
      'passwordFilePath': password_file_path
    }
```
If 'url' variable is string with jdbc scheme, sqoop command is valid and will be executed.
But without jdbc scheme, sqoop command will be failed.

Reply:
Thanks for the detailed explanation! The change looks good for now.

Maybe this can be improved from the UI side for long-term when we work on revamping the importer as a roadmap item. @bjornalm